app/controllers/opportunity_matches_controller.rb

```diff
@@ -75,6 +75,38 @@ def create_match_attributes(client_id)
     }
   end
 
+  def prioritized_column_data
+    @prioritized_column_data ||= Client.prioritized_columns_data
+  end
+
+  def prioritized_column_labels
+    [].tap do |result|
+      @opportunity.match_route.prioritized_client_columns.map(&:to_sym).each do |column|
+        column_data = prioritized_column_data[column]
+        next if column_data.blank?
+        next if column_data[:display_check].present? && send(column_data[:display_check]) == false
+
+        title = column_data[:title]
+        title = column_data[:alt_title] if column_data[:alt_title].present?
```

Comment:
This structure seems somewhat more complicated than it probably needs to be.  There are only 3 alt-titles.  I'll drop some notes below, but I think we can just standardize `title`